src/Module/Phiremock.php

```diff
@@ -122,4 +123,10 @@ public function grabRequestsMadeToRemoteService(RequestBuilder $builder)
     {
         return $this->phiremock->listExecutions($builder);
     }
+
+    public function setScenarioState(string $name, string $state): void
```

Comment:
[change request] Please remove the type hints. Version 1 of this extension requires php5.6 as minimum.

Reply:
I removed the type hints. Check again please.

P.S. This is my first PR to open-source projects ;)

Reply:
Cool.
Thank you very much for your help and your PR.